pkg/reconciler/kubernetes/tektonhub/tektonhub.go

```diff
@@ -75,8 +77,10 @@ const (
 	dbInstallerSet          = "DbInstallerSet"
 	dbMigrationInstallerSet = "DbMigrationInstallerSet"
 	apiInstallerSet         = "ApiInstallerSet"
+	uiInstallerSet          = "UiInstallerSet"
```

Comment:
where are this used?
we should name as `HubApiInstallerSet` and similar

Reply:
while storing the name of installer set in tekton hub cr
I think it should be fine keeping it the same because 

when we do `k get tektonhub hub -oyaml` at the end we'll see the following map under `hubInstallerSets`
```
  hubInstallerSets:
    ApiInstallerSet: tekton-hub-api-xlnxs
    DbInstallerSet: tekton-hub-db-pbfpd
    DbMigrationInstallerSet: tekton-hub-db-migration-9knfw
    UiInstallerSet: tekton-hub-ui-qfpwp
```